src/common/telemetry/src/logging.rs

```diff
@@ -68,6 +68,9 @@ pub struct LoggingOptions {
 
     /// The logging options of slow query.
     pub slow_query: SlowQueryOptions,
+
+    /// The maximum number of log files set by default.
+    pub max_log_files: usize,
```

Comment:
Can we expose this option in [configurtion file's log section](https://github.com/GreptimeTeam/greptimedb/blob/aa03d3b11ccff692ad348cd113da46a7e758b527/config/standalone.example.toml#L604)?